core/src/test/java/com/es/core/model/phone/JdbcPhoneDaoIntTest.java

```diff
@@ -0,0 +1,69 @@
+package com.es.core.model.phone;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.Resource;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+@ContextConfiguration("classpath:context/testApplicationContext-core.xml")
+@Transactional
+@RunWith(SpringJUnit4ClassRunner.class)
+public class JdbcPhoneDaoIntTest {
+
+    @Resource
+    private JdbcPhoneDao jdbcPhoneDao;
+
+    @Test
+    public void testGetPhone() {
+        Optional<Phone> phone = jdbcPhoneDao.get(1000L);
+
+        assertTrue(phone.isPresent());
```

Comment:
Will be better to check id, then isPresent. And your tests depends on demodata-phones.sql. Tests should be independent. If we add new property to phone you should change all inserts. You can create new phone, save them and then get this phone and check